samples/secure-mcp-sse-server/README.md

```diff
@@ -12,6 +12,15 @@ Start the application in the dev mode:
 ./mvnw quarkus:dev
 ```
 
+It also starts a https://quarkus.io/guides/security-openid-connect-dev-services#dev-services-for-keycloak[Keycloak Dev Services container].
+
+[NOTE]
```

Comment:
@sberyozkin I'm afraid that this is not a valid md either ;-)

Reply:
ping @sberyozkin

Reply:
Oops, sorry @mkouba, I missed, yes, this one is related to the one on main that was done before #251, so I thought I'd follow with the #251 backport for the 1.1.1 branch, does it work ?